BitwardenShared/UI/Platform/Application/AppProcessor.swift

```diff
@@ -115,22 +115,13 @@ public class AppProcessor {
     /// - Parameter url: The deep link URL to handle.
     ///
     public func openUrl(_ url: URL) async {
-        guard url.scheme?.isOtpAuthScheme == true else { return }
+        guard let scheme = url.scheme else { return }
 
-        guard let otpAuthModel = OTPAuthModel(otpAuthKey: url.absoluteString) else {
-            coordinator?.showAlert(.defaultAlert(title: Localizations.anErrorHasOccurred))
-            return
+        if scheme == "bitwarden" {
```

Comment:
⛏️ Can `"bitwarden"` scheme be moved to a constant?

Reply:
Pulled this into a constant in the latest. 👍